.bazelrc

```diff
@@ -1,7 +1,13 @@
-build --host_javabase=@local_jdk//:jdk
 build --host_platform=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host
 build --compilation_mode opt
 run --compilation_mode opt
+
+build --java_runtime_version=11
+build --java_language_version=11
+build --tool_java_language_version=11
+build --tool_java_runtime_version=11
+build --experimental_strict_java_deps=off
```

Comment:
`bazel` complains otherwise that "//jvm-batching:jar" is missing from the direct dependencies of "//jvm-types:jar" even though the dependency is listed [there](https://github.com/tweag/inline-java/blob/5bb2fea980a30cf6630b2481539db031e8e8d7fb/jvm-types/BUILD.bazel#L10).